### PR TITLE
Symfony 4

### PR DESCRIPTION
**Pull request by chregu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,6 +40,9 @@ matrix:
     - php: 7.1
       env:
         - SYMFONY_VERSION='3.2.*'
```
Comment (Tobion) [inline .travis.yml]: symfony 3.1 and 3.2 are outdated already and non-lts. so i'd say we don't need to tests against those anymore but instead 3.4 which is lts

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ public function getConfigTreeBuilder()
                 ->end()
                 ->scalarNode('api_secret')
                     ->info('Api-secret value from user registration.')
```
Comment (Tobion) [inline src/DependencyInjection/Configuration.php]: please update the info if this is not meant to be used anymore.

File: tests/Functional/Fixtures/AppKernel.php [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace Rokka\RokkaClientBundle\Tests\Functional\Fixtures;
```
Comment (dbu) [inline tests/Functional/Fixtures/AppKernel.php:3]: @Tobion how does this work with both older and newer symfony tests? or should i make this class without namespace and then make composer dev classmap find it?

Comment (Tobion) [reply tests/Functional/Fixtures/AppKernel.php:3]: I'd keep it like this and create a `class_alias` for AppKernel to the namespaced class

File: .travis.yml (hunk 1 above)
Comment (chregu) [reply .travis.yml]: I added 3.4 to the tests (as default for all php versions)
3.2 is officially still security supported for a month. But yes, we could remove those two.

**Comment by chregu** [comment]
Thanks a lot for your help everyone

**Pull request merged**